### DISCUSSION TITLE
[ideas] "Dry-run" option for `tar_prune()`

### DISCUSSION THREAD

**Discussion by mglev1n** [ropensci/targets#1090] [Ideas]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

The `tar_prune()` function is a somewhat dangerous intermediate between `tar_destroy()` (which deletes the entire target store, and `tar_delete()` (which deletes only the specified target). While `tar_prune()` is designed to remove unused targets from the target store, it's not obvious to the user which targets/files will actually be deleted. I wonder if it would be possible to include a "dry-run" option that would print a list of unused targets that will be removed, or otherwise indicate to the user which targets will be destroyed. Apologies if this functionality is already contained within another function/can be generated easily using `tar_meta()`.

**Comment by wlandau** [comment]
Yeah, I think it's time. Just added `tar_prune_list()` for this.
